Tests/Data/RichardsMechanics/RichardsFlow_2d_small.prj

```diff
@@ -13,67 +13,103 @@
-            <solid_density>rho_sr</solid_density>
             <process_variables>
                 <displacement>displacement</displacement>
                 <pressure>pressure</pressure>
             </process_variables>
-            <material_property>
-                <fluid>
-                    <density>
-                        <type>Constant</type>
-                        <value> 1 </value>
-                    </density>
-                    <viscosity>
-                        <type>Constant</type>
-                        <value> 1.e-3 </value>
-                    </viscosity>
-                </fluid>
-                <porous_medium>
-                    <porous_medium id="0">
-                        <permeability>
-                            <permeability_tensor_entries>k</permeability_tensor_entries>
-                            <type>Constant</type>
-                        </permeability>
-                        <porosity>
-                            <type>Constant</type>
-                            <porosity_parameter>phi</porosity_parameter>
-                        </porosity>
-                        <storage>
-                            <type>Constant</type>
-                            <value> 0.0 </value>
-                        </storage>
-                        <capillary_pressure>
-                            <type>vanGenuchten</type>
-                            <m>0.789029535864979</m>
-                            <pc_max>36333.30</pc_max>
-                            <pd>3633.33</pd>
-                            <smax>0.95</smax>
-                            <sr>0.1689</sr>
-                        </capillary_pressure>
-                        <relative_permeability>
-                            <type>WettingPhaseVanGenuchten</type>
-                            <krel_min>1e-12</krel_min>
-                            <m>0.789029535864979</m>
-                            <smax>0.95</smax>
-                            <sr>0.1689</sr>
-                        </relative_permeability>
-                    </porous_medium>
-                </porous_medium>
-            </material_property>
             <secondary_variables>
                 <secondary_variable internal_name="sigma" output_name="sigma"/>
                 <secondary_variable internal_name="epsilon" output_name="epsilon"/>
                 <secondary_variable internal_name="velocity" output_name="velocity"/>
                 <secondary_variable internal_name="saturation" output_name="saturation"/>
             </secondary_variables>
             <specific_body_force>0 -9.81</specific_body_force>
-            <temperature> temp </temperature>
         </process>
     </processes>
+    <media>
+        <medium>
+            <phases>
+                <phase>
+                    <type>AqueousLiquid</type>
+                    <properties>
+                        <property>
+                            <name>bulk_modulus</name>
```

Comment:
The only thing I'd like to discuss here is wether we agree to use either bulk moduli _K_ or compressibilities _beta=1/K_ for fluids and/or for solid phases. Personally, I like the compressibility approach, but this is just a personal preference. But, I am worried that it may look inconsistent (in fact, it is) for a user, since some processes require _K_, some require _beta_. Like many other things in MPL, we should discuss that in the near future.

Reply:
Let's collect these things for the upcoming community meeting: https://github.com/ufz/ogs/wiki/Community-meeting-2019